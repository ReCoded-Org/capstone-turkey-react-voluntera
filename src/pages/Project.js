import ApplicantList from '../components/ApplicantList';
import ProjectContent from '../components/ProjectContent';

function Project() {
  return (
    <div className="container mx-auto px-8 py-6">
      <ProjectContent />
      <ApplicantList />
    </div>
  );
}

export default Project;
