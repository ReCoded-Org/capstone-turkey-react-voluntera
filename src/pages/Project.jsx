import ApplicantList from '../components/ApplicantList';
import ProjectContent from '../components/ProjectContent';

const projectimage = '../assets/images/project.jpg';

function Project() {
  const applicants = [
    {
      id: '1',
      name: 'Emily Jonsen',
      position: 'Manager at Company',
      applicationStatus: 'accepted',
    },
    {
      id: '2',
      name: 'Sara Bush',
      position: 'Author at Company',
      applicationStatus: 'rejected',
    },
    {
      id: '3',
      name: 'Hilary Amber',
      position: 'Cashier at Company',
      applicationStatus: 'null',
    },
  ];
  return (
    <div className="container mx-auto px-8 py-6">
      <ProjectContent projectimg={projectimage} />
      <ApplicantList applicants={applicants} />
    </div>
  );
}

export default Project;
