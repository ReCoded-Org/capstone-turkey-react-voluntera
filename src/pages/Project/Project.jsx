import ApplicantList from '../../components/ApplicantList';
import ProjectContent from '../../components/ProjectContent';

function Project() {
  const applicants = [
    {
      id: '1',
      name: 'Emily Jonsen',
      position: 'Manager at Company',
      applicationStatus: 'accepted',
    },
    {
      id: '2',
      name: 'Sara Bush',
      position: 'Author at Company',
      applicationStatus: 'rejected',
    },
    {
      id: '3',
      name: 'Hilary Amber',
      position: 'Cashier at Company',
      applicationStatus: 'null',
    },
  ];
  const project = [
    {
      id: '1',
      name: 'Clean Environment',
      description:
        ' You can load project details such as the project name, classification, and type, as well as the start and end dates, to existing and new projects. You load project details to existing projects by using a data file with the same format as the ProjectDetails.csv data load file',
      date: 'Mar 12 2020',
      skills: ['Public speaking', 'Teaching', ' Good communication'],
      address: '16 Woodlawn Terrace, Waterbury, ct, 6310 United States',
      time: 'Part-Time',
      applicants: '495',
      image:
        'https://retaintechnologies.com/wp-content/uploads/2020/04/Project-Management-Mantenimiento-1.jpg',
    },
  ];
  return (
    <div className="container mx-auto px-8 py-6">
      <ProjectContent project={project} />
      <ApplicantList applicants={applicants} />
    </div>
  );
}

export default Project;
