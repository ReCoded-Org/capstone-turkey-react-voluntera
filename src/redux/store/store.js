import { configureStore } from '@reduxjs/toolkit';
import signupVolReducer from '../slices/signUpSlice';
import signupOrgReducer from '../slices/signUpOrgSlice';
import projectsReducer from '../slices/projectsSlice';
import postsReducer from '../slices/postsSlice';

const store = configureStore({
  reducer: {
    signupVolunteer: signupVolReducer,
    signupOrganization: signupOrgReducer,
    projects: projectsReducer,
    posts: postsReducer,
  },
});

export default store;
