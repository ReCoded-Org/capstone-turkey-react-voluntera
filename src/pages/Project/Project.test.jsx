import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';

import Project from './Project';
import store from '../../redux/store/store';

test('render the project page', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Project />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
