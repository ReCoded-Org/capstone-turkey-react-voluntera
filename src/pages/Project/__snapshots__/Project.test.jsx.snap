// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the project page 1`] = `
<div
  className="container mx-auto px-8 py-6"
/>
`;
